hvac/api/auth_methods/aws.py

```diff
@@ -146,34 +146,52 @@ def configure_identity_integration(
             select full_arn and then delete and recreate the IAM role, Vault won't be aware and any identity aliases
             set up for the role name will still be valid
         :type iam_alias: str | unicode
+        :param iam_metadata: The metadata to include on the token returned by the login endpoint.
+            This metadata will be added to both audit logs, and on the iam_alias. By default, it includes account_id 
+            and auth_type.Additionally, canonical_arn, client_arn, client_user_id, inferred_aws_region, inferred_entity_id, 
+            and inferred_entity_type are available. To include no metadata, set to "" via the CLI or [] via the API. 
+            To use only particular fields, select the explicit fields. To restore to defaults, send only a field of default. 
+            Only select fields that will have a low rate of change for your iam_alias because each change triggers a storage 
```

Comment:
```suggestion
            This metadata will be added to both audit logs, and on the ``iam_alias``. By default, it includes ``account_id`` 
            and ``auth_type``. Additionally, ``canonical_arn``, ``client_arn``, ``client_user_id``, ``inferred_aws_region``, ``inferred_entity_id``, 
            and ``inferred_entity_type`` are available. To include no metadata, set to an empty list ``[]``. 
            To use only particular fields, select the explicit fields. To restore to defaults, send only a field of ``default``. 
            Only select fields that will have a low rate of change for your ``iam_alias`` because each change triggers a storage 
```
nits and formatting